object_detection/pipeline/pipelinerun.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  name: object-detection-pipeline-run
```

Comment:
use generateName like this https://github.com/openshift/pipelines-tutorial/blob/master/pipeline/pipelinerun.yaml#L4